### ISSUE TITLE
[RA] Isolate new auto targeting vs structures towards Return Fire/Hold Fire

### ISSUE THREAD

**Issue by SoScared** [OpenRA/OpenRA#14001]
I wanted to wait on this issue ticket until a later time with this playtest cycle as I hoped more players would have the experience with the unit stances, particularly the broader community affiliated with big team games, which represents the bedrock of the online and multiplayer community. Unfortunately as with every previous playtest cycle in the past this particular sector won't have much exposure of the mod's changes, if any at all, until the offial release has been shipped. In contrast with most previous releases however, the RA mod has with it planned features that for the most part carry no real foresight as to how it will be experienced by the broader community, leaving most the discussion left to guesswork.

The last time we had a halfway similar event was with release 2013.12.23 with the introduction of the 16c0  Con.Yard build radius. For most people this change came aboard with minimal preparation but still it was anticipated as there was a strong demand to curb infinite basewalking for a long period of time. https://github.com/OpenRA/OpenRA/issues/2156

The problem and potential pitfall that _I_ see with the auto targeting is pretty simple and has nothing to do with balancing, unit control or anything like that. As I see it, the totality of the negative side of the feature's effect experienced by the players within the RA mod will _by far_ come from the silent majority – the casual online playerbase, for the most part frequently tied up with massive team games and players off-line content with playing against the A.I. The changing back and forth, on and off the "building-attack-stance" (these players have absolutely no interest in 'Defensive' vs 'Attack Anything' behavior) is how it's gonna translate in the mind of these particular lot of players. A prerequisite game mechanic to be executed in order to finish off an enemy and close out the game. This issue get's wildly proportional with the increasing number of enemies and for the online team games you have by default scenarios with great amount of units swarming through huge fields of buildings waiting to be turned on by the player only to have it be done again with the next wave of units and/or with the next move should the player decide to switch back to the default stance. The constant going back to the command bar or using hotkeys to execute these commands is a simple matter to the prominent named community members but for the pool of players actually keeping OpenRA alive it's gonna be a whole different ballgame.

Besides the implementation from a simple technical standpoint you also have the visual side of it where you'll see a big group of units suddenly "turns on" and starts shooting at every building around them in a quasi-synchronized way per unit-type.

Having players adapt to new features with new ways to control units is something I believe I understand to a certain degree and I do absolutely understand the sentiment of requiring players to learn new sets of tools to further a positive agenda but in this particular case I believe very strongly the response will be that of disappointment once the reality of the feature kicks in with the above described scenarios, which according to my guesstimate represents over 90% of how the RA mod is experienced by the totality of RA's playerbase. It's simply just not user friendly and for many of these players, who don't give a second thought about ref placements, build-orders, micro or executing advanced strategy, just won't understand the added inconvenience seemingly coming out of nowhere for reasons they won't understand. Not because they're disinterested in the game's development but simply because it directly put's a wedge in-between the players and them having plain _simple_ fun – and I emphasize 'simple' because as of now there's an excellent 'flat scale' ranging from experiencing the game from the most basic level to the smaller core of expert players taking advantage of game mechanics unavailable to new players, simply because of level of skill. In my view the foundational 'simple' is the first step and sole reason for why most people are coming on to OpenRA and in many cases play the mods endlessly to eventually becoming a part of an active community.

The feature cuts deep into the identity of the OpenRA project itself, constantly struggling to join two polar opposites which conserves the spirit of the archaic C&C Red Alert versus improvements brought on board from its successors. Every deviation has its existence ideally rooted in necessity of making the game fun and playable with emphasis on skirmish and by extension, the online multiplayer mode. If we wanted to experience the original as-is, particularly the campaigns, we only have to look towards the abandon-ware. The 'avoid-targeting-buildings' stance emulates the experience drawn from the original game where you could overcome the enemy with your forces, camp its base and take over all the buildings with your engineer. In fact, growing up with this game, I did this religiously with every mission just for the role-play of utterly dominating and humiliating the enemy and if possible, beat them with their own units. It was certainly a fond part of the original game but as-is this game mechanic doesn't translate well over to the RA mod's skirmish. The mod now having availed almost every game asset to the player, popularized online multiplayer and "inflated" the game to enormous levels when it comes to the count of units and structures in any given game, particularly big team games, fully eclipses the role of the original targeting to the point where it lines up with other archaic features that made sense at the time but currently just serve to limit the experience.

The above point simply never came across properly in the feature's playtest discussion, one root reason being that the game almost exclusively is being analysed publicly from a competitive viewpoint. Usually there's an approach from both sides that eventually falls on a solution that everyone can relate to. Granted, regarding balancing I've been a sort of rogue mediator the past two  release cycles but for other feature related changes, AFAIK there are very few implementations that fell short with the playing community. In this particular case the discussion was also polarized right off the bat and a clear-headed discussion about what that change would mean for the game as a whole was simply not possible. 

The juicy reward following this feature makes itself evident in the mods' campaigns. I had a run through with the RA playtest campaign missions and it's definately an added 'blast from the past' experience with camping the AI in order to gain yourself a new base with the enemy tech but it also underscores why the feature can be expected to work differently for the missions vs skirmish as the mission A.I. bases are usually both limited and static. For the missions there's a very strong argument for bringing this feature onboard but the question also then falls on wether or not to formalize the feature across all play modes as the gamerules deviation between the missions and skirmish are currently very few and with every addition to that list would be a noticeable step in widening the gap between the two game modes.

The idea proposed as mentioned in the title would have the 'no-targeting-buildings' left for the ReturnFire/HoldFire stances, meaning it's a feature the player turns _on_ in order to wipe out everything except unarmed buildings. Specifically, A-moving with AttackAnything/Defensive would target everything in its path as default but with ReturnFire/HoldFire would target only units and defensive structures, making way of a potential take-over with engineers or just for plain old fun, if wanted. The point is this feature from that point is an addition, enriching the experience in a way that would immediately be beneficial for both missions _and_ casual/competitive play. It also falls neatly within the stance categories with a two-for-two and for any concerns regarding unwanted side-effects with AttackAnything vs Defense stance, would be gone.

One important thing pointed out about this feature is that it's been in the works for a very long time, with an early attempt to include the feature in 2013 (#3337) but because of issues with the A.I. it was shelved until the surrounding issues was resolved. Couple the hard work of making the feature possible with the stated development goals for OpenRA it should follow that this feature makes its way into OpenRA's flagship mod. However it's my conviction the feature would be celebrated as a feature added on top of the current workings of the game, providing a link back to the original game for the missions and an exiting tool for players to explore with both skirmish and online multiplayer.

**Comment by AoAGeneral** [comment]
Main thing about the silent majority though is those who have played the original CNC95 and RA95 is this was the base game mechanic where they ignored structures and shot the units instead. This was the mechanic that was in ORA when I first joined many years ago. (Around the 2014 of Feb 4th.) It was then switched around in the next release which caught me by surprise but learned to adapt around it.

Currently as of right now im looking into getting additional feedback on the stances itself with some test games since the playtest has been pretty barren for a long time.

**Comment by pchote** [comment]
Like @AoAGeneral said: we shipped this with the TD mod for nearly a year and a half, although the dates are a bit off.  It was added in release-20120504 and removed again in release-20130915 after the AI issue was discovered after adding it to the RA mod.

> the spirit of the archaic C&C Red Alert

This feature was not limited to the original RA.  This behaviour persisted from C&C1 through to RA3, and is as much part of the C&C gameplay formula as harvesting or engineers.

> The changing back and forth, on and off the "building-attack-stance" (these players have absolutely no interest in 'Defensive' vs 'Attack Anything' behavior) is how it's gonna translate in the mind of these particular lot of players.

I expect that many casual players won't notice this at all.  My experience playing the game with office mates, watching youtube videos of new players, and my own experience from playing the original games all indicate that most casual players prefer to manually target buildings, and do not take advantage of stances or use attack-move for base clearing purposes (a lot of them don't use attack move *at all* 🤷‍♂️)

**Comment by GraionDilach** [comment]
>The 'avoid-targeting-buildings' stance emulates the experience drawn from the original game where you could overcome the enemy with your forces, camp its base and take over all the buildings with your engineer.

Everything drawn from this point onwards sounds wrong or overstated. The reason why passive targets weren't auto-acquired in the legacy games was to aid the players via allowing the units to retaliate without explicit orders, to **decrease the micro, attention and unit losses required to level a base**. So you won't lose your units just because they decided to attack the lab on their own and becoming sitting ducks for the enemy's newly trained unit X. Also note that all the traditional C&Cs (IIRC Gen was the first one which didn't do this for all factions but only at US and notsure on the followups) had **destroyed building survivors** which would also increase the micro requirement of the player and would require more attention with the proposed stance setup, considering that according to your own testing with RevealOnDeath, there is a frame difference between destroy and spawn, which would automatically place such survivors to a beneficial situation since by the time they would pop up, the attackers already autoacquired the next powerplant.

In fact, the whole point of waypoints were that you sacrificed unit awareness and attention while taking the risk of not noticing/outright ignoring such a retaliation. And I don't see why should out-of-all, the defend stance sacrifice this retaliation-benefit. No balance change can justify the sacrifice of this bonus.

Besides some players bemoaning that this "ruins their superspecial and unique playstyle, so this is bad", I haven't seen any reasoning on the opposing side of the coin. It shouldn't be the developers' issue that *these people rely on an abuse* to achieve their playstyle. If there are bugs with queuing waypoints, that needs to be fixed with priority, and these vocal players should adopt relying on waypointing the buildings on a 1-by-1 basis.

**Comment by Omnomon** [comment]
I side with pchote in the sense that the stance system is an overstated problem. Once it's in the game, people will adapt to the new attack-move logic. Having the new attack-move logic being the default stance (as SoS mentions) or it in the game at all (as other people have mentioned) is purely subjective and at the end of the day, it's up to the devs to decide what makes the most sense. 

However, I want to point several things out that makes **no sense** from the user's point of view:

1. What you are currently trying to ship would be better implemented as a separate command (i.e, a single keypress + mouseclick) rather than a stance command (two keypresses + mouseclick, **along with additional keypresses to change stances** ). 

As a player, this is what I see: 

> Attack Anything ---> Attack move --> Attack all enemy units
> 
> Defense Stance --> Attack move --> Attack only defensive structures and other units 
> Hold Fire Stance --> Attack move --> Attack only defensive structures and other units 
> Return Fire Stance --> Attack move --> Attack only defensive structures and other units 

Wouldn't it be simpler and more efficient to have:

> Attack move --> Attack Everything 
> [insert button here] move --> Attack only defensive structures and other units

Forcing the players to execute multiple commands to have the units get access to a certain behavior is not going to make people want to use it, no matter how good the other stances are. 

2. The stance system does not work in unison with the command system. 

Sometimes, players might want to order units to _"Go to point A, then attack anything_" or "_Go to point B, then hold position_." This is where stances may come in handy. However, a user will **never** want to order a command "_Until I say otherwise, hold your fire_." In addition, in-game situations will rarely, if ever, call for the player to order their units "_When you're standing still, hold your fire."_ In addition, in the current release, the most common army command, the Attack-Move, is the exact same thing for all 4 stances. Even in the upcoming release, where the Attack-Move logic is changed between a few stances, 3 out of the 4 stances still share the same logic. I fail to see the purpose of stances with regards to the command system. 
 
In other words, **persistant** stances do not make any sense when combined with commands. The options that the current stance system provides resemble modern "rules of engagement" -- why is there a system for this when the goal is to just beat your opponent?  Now this leads to the circular debate of  "casual players aren't going to use the stance system anyways." --"why is the stance system even in the game in the first place" --> "we wanted to make openra resemble the original more" --> "casual players aren't going to use the stance system anyways". 

In my honest opinion, with regards to the stance system, I highly suggest you diverge from your project statement and cater to the player experience here. Right now, the stance system is in a weird state of limbo where "stances are useful in this niche case"  and "stances are completely useless." If you want to enhance the stance system, make it more usable and user friendly. If you're going to keep it in it's current state, get rid of it entirely.

**Comment by AoAGeneral** [comment]
The stance system works as intended from what ive been playing with. From the other players im currently waiting on some test games or replay examples of the issues that was mentioned.

Hold fire is an important need. While attack move performs the method as you say, if you have V2 on hold fire and do a move command rather then attack move then they won't fire even when you hit the S key or reach their destination. Same applies to return fire. So the different commands do have their different functions unless using attack move. But its a common movement command with everyone which is why seperating your V2 from a control group is the way to go when on hold fire.

This also applies to APCs in TD having hold fire and moving around so they don't "Alert" your opponent when being sneaky.

**Comment by pchote** [comment]
> Wouldn't it be simpler and more efficient to have:
>
>> Attack move --> Attack Everything 
>> [insert button here] move --> Attack only defensive structures and other units

This is pretty much the attack move vs assault move distinction that I mentioned above.  Except that attack move works as it does in the current playtest (obeying stances), while assault move will target any units or buildings within range.

**Comment by Omnomon** [comment]
> if you have V2 on hold fire and do a move command rather then attack move then they won't fire even when you hit the S key or reach their destination
This is what I meant when I said "rarely, if ever." There are multiple ways to stop V2's from firing on stuff you don't want attacked, and sneaking an APC in your opponents base is not  primarily dependent on the APC holding its fire.  My point with the stance system is that it's in a very awkward place right now, and the new attack-move logic makes the stance system even more awkward.

>  This is pretty much the attack move vs assault move distinction that I mentioned above. Except that attack move works as it does in the current playtest (obeying stances), while assault move will target any units or buildings within range.

I interpreted your post regarding this as a "last resort" of sorts, but I believe this change would appease the vast majority of the current community and future players. I hope you decide to implement this alongside the attack-priority changes to the stance system. As a sidenote, I hope that this means stances can now be modded through YAML as well?

**Comment by pchote** [comment]
@Omnomon: would you be happy to live with requiring the control key to activate the attack-anything / assault behaviour when issuing an attack-move order?  This would be a relatively straightforward thing to implement compared with implementing a new order type, new button artwork, new hotkey, etc.

**Comment by AoAGeneral** [comment]
I am unaware of the multiple ways to keep V2s from firing. I simply use hold fire as its much easier. If there is an easier way that I am unaware of would be interested in hearing it.

As for the APC it is because if it fires at a structure it alerts the player "Base under attack" which is primarily not what you want to do.

**Comment by Smittytron** [comment]
Another way would be for a one-key toggle that switches between Defensive stance and Attack-anything stance. I favor this method because it reduces the amount of keys used. That said the 'assault move' would work just fine and I have no objections there.

**Comment by Omnomon** [comment]
@AoAGeneral If you want to learn, you can go ask somewhere else; i will not derail this thread further by indulging your tangent. 

> would you be happy to live with requiring the control key to activate the attack-anything / assault behaviour when issuing an attack-move order?

This would be a very welcome addition. My only question is that wouldn't this interfere with the current force fire mechanic?

**Comment by reaperrr** [comment]
Personally, as someone who mostly plays sp missions and skirmish, I've always found the old default "attack everything automatically, even buildings" behavior pre-AutoTarget-refactor pretty annoying, so to me what we currently have is better than what we previously had. In my opinion the current command bar + some "AssaultMove" should be good enough to cover those cases where a different behavior is desired (+maybe  setting a custom default stance at the factory, if that's still on the agenda).

We'll never be able to make everyone 100% happy without exception anyway, not matter what we do.

**Comment by rautamiekka** [comment]
^
>We'll never be able to make everyone 100% happy without exception anyway, not matter what we do.

Very true, but customization options bring closer to that, and it shouldn't be much of a problem to implement either.

**Comment by pchote** [comment]
> My only question is that wouldn't this interfere with the current force fire mechanic?

I encourage you (and everyone else) to try this prototype and see:
* [macOS disk image](https://www.dropbox.com/s/k1hoi7pde6dex3n/OpenRA-devtest-20170909.dmg?dl=0)
* [.deb package](https://www.dropbox.com/s/ovr7fzxkpz690b1/openra_devtest.20170909_all.deb?dl=0)
* [Windows installer](https://www.dropbox.com/s/h04h62wccxdwh4n/OpenRA-devtest-20170909.exe?dl=0)

**Comment by Omnomon** [comment]
The one "bug" with the Assault-move is that units will execute a 1c0 move before finding a target to attack. I think this is just a byproduct of how the A-move is currently calculated, but it is quite noticeable, especially if your unit is already attacking something. Other than that, I think this would be a welcome addition to the game, especially if you allow players to rebind this in the Settings tab.

**Comment by SoScared** [comment]
Adding relevant discussion here:

[18:09:39] Sep 9 2017

```
<pchote> SoScared: if i'm going to add this assault-move thing then we're going to need another playtest
<pchote> so we might as well include that, Smitty_'s paratrooper announcement, and the hijacker fix
<SoScared> pretty much outside my field of vision, but sure
<SoScared> wait. assault move?
<pchote> SoScared: read below your wall of text
<SoScared> was gonna wait a few days for that
<pchote> you may want to move that forward
<SoScared> eh
<SoScared>  well, in short the counter-arguments rests on the original game  mechanics, imo fully irrellevant to the current mod, at least in the way  described by AoA and Grayon.
<pchote> SoScared: in the same way that the forward-arguments rest on the current game balance, which are not immutable
<SoScared>  the rest, pointing to 'people will just have to adjust' argument rests  as far as I can see on wether or not my intuition with how the casual  player will relate to the new game mechanic
<SoScared> it then slides onto irrelevant competetive discussion (as originally unintended) which I have 0 interest to follow up.
<SoScared>  the issue ticket itself concerns the perception of a sector of players  not involved with the active community or contributors/developers
<SoScared> in short, so far the issue discussion looks like going nowhere as far as the focus of the issue is concerned
<SoScared> I'll just keep an eye for the time being
<pchote> did you miss the part where I suggested a compromise?
<pchote> which I have now implemented
<SoScared>  if it requires people to tap buttons each time they want to clear out  stuff by defaut, I'd guess it doesn't really touch upon the concerns of  the issue ticket
```

Following that a suggestion of me posting the issue ticket as a big tantrum against change threw me off quick and made me very angry, especially after having put a lot of thought and patience with my concerns explained in excruciating detail with the OP and especially given that the topic in general concerns itself with guesswork.

Look, my entire argument rests on whether or not the feature will cripple the enthusiasm among players of which the feature will affect the most and who has the least say on this matter. The 'players will adapt' mantra don't really mean anything if result is experienced as dissatisfaction. Players don't adapt if they stop playing and the fact that this scenario isn't discussed at all as a possibility makes me equally concerned with how the contributors and developers will react _if_ there is truth to what I'm saying. If there's one thing this community has proven more than once it's that it doesn't handle controversy and turmoil very well.

I'm more than happy to be wrong on topics and ideas I bring up and I don't mind having my points be overlooked but as I've literally based my entire modus operandi on speaking out strongly for what I personally believe in regardless of it aligning with the agenda of the developers or with the community's greatest loudmouths. With the polarized discussion surrounding this issue my say on the matter is undoubtedly affected by the position of my argument instead of the argument itself.

As  mentioned by me on IRC the discussion on this assault-move patching, tho possibly interesting in and off itself, doesn't really intersect well with what the original topic concerns itself about.

**Comment by Omnomon** [comment]
I understand your complaint and that you have the casual player's best interests at hand, but I could easily rewrite your argument to support making the new Defense A-move the default stance. No matter how good or sincere your reasons are, what you are trying to get changed is purely subjective; some people like the new default auto-targeting, some people hate it.  No matter what Pchote decides to do, half of the community is going to hate the changes, and the other half is going to applaud his changes.  

We can agree to disagree on how this logic is implemented and how it should be set up. The best solution on the table right now is combining the Assault-move alongside the Attack-move and stance changes to give the player multiple, viable options to control their army.

**Comment by SoScared** [comment]
> I could easily rewrite your argument to support making the new Defense A-move the default stance.

Then do it, don't just say it and then dismiss my clam based on your stated ability to form an opinion.

Btw your statement is nonsensical regarding that people will hate _not_ having the default auto-targeting altered when it's active in the current release without anyone upset about it.

**Comment by Omnomon** [comment]
> Then do it, don't just say it and then dismiss my argument based on your stated ability to form an opinion. Pretty lazy and frankly dishonest.

The last time I tried to express my opinion through walls of texts, I got into a pointless argument with AoA over something stupid that I don't want to remember. If you really want to have a never-ending, moot discussion, we can do it somewhere else. 

I'm not trying to offend you or dismiss your argument; I am trying to point out that there are proponents and opponents for the topic that you are arguing about. You want people to turn on these new features, other people want these features to be the default.. Hell, we're probably on the same damn side when it comes to stances. We all have opinions, but no matter how well-written they are, they're still just opinions -- nothing will ever get done if the only thing we do is write walls of biased, opinionated texts because no one is willing to listen and bend their stance. 

The one fact that we can all agree on is that we want this game to be better. The only way we're going to get there is if we cooperate and compromise with each other rather than creating some flaming thread. Pchote decided to compromise, and it'd be nice if the players would be willing to extend him that courtesy as well.

**Comment by SoScared** [comment]
With all due respect I'm not AoA so please do not bring your issues thereof over to me. I don't avoid points or start pulling out new ones to fit the moment.

There's no way you've offended me but for I get pretty sharp vs what I perceive as statements crafted to glaze over the core of an argument and I seldom address it without coming across as adversarial. I have no interest in creating a debacle and I have no expectations with anyone believing or agreeing to what I'm saying.

**Comment by Omnomon** [comment]
Fair enough. Shorter responses inevitable glaze over and summarize the core content of any argument, and I know stuff gets lost through when comments are abridged. I am just simply tired of constructing walls of text for any purpose whatsoever.

**Comment by GraionDilach** [comment]
Well, I'll be TLDR then and just raise a single question, since this is all down it comes about, and I feel you've ignored this one/called it irrelevant in the IRC log fragment.

Why should I micro my stuff to concentrate on the actual threats within an enemy base if I'm on the Defend stance?

**Comment by pchote** [comment]
@SoScared: It's a straightforward question IMO, which deserves a straightforward answer.

I can start by giving a specific example of one of the downsides of the old behaviour: units auto-targeting buildings has lead to the rather perverse situation where players use barracks as offensive units, because they act as very efficient distractors to defending units.  This makes no sense from a logical or gameplay perspective.

**Comment by GraionDilach** [comment]
@SoScared This isn't about balance, but about a core gameplay design element. You're doing a mistake if you're analyzing it strictly within RA. You're forcing to get through a core gameplay design aspect based on some players on a single mod - something which should be consistent between all official projects atleast. And I can go on that this is against OpenRA's D2k and TS aims thenafter.

**Comment by SoScared** [comment]
As demonstrated in yesterday's stream by LorryDriver (vod link) one can observe how basepushes quickly gets crowded with buildings in the front-line and players manually targeting buildings with units and defensive structures. This micro managing costing extra clicks and attention in turn supported bigger and more passive army blobs, particularly infantry. Now this was demonstrated by some of the communities fastest players in a 1v1 settings. Expand that scenario over to a team game with 10 players playing casually and it shouldn't IMHO take too much an imagination to see the potential mess that could be taking place.

The competitive players having the ability to deal with this game mechanic to limit its effect won't bear the full burden of the potential fallout but the casual player I can imagine certainly would.

**Comment by SoScared** [comment]
@GraionDilach I can appreciate your concern regarding other mods but I've never pretended to speak on behalf of other mods besides RA - hence the [RA] in the title of the issue ticket.

**Comment by rautamiekka** [comment]
Is making it customizable not an option ?

**Comment by GraionDilach** [comment]
@SoScared Timestamp, please? I don't have the time to go through 4 hours nor I won't try to randomly wander in it. At this stage I'm gonna assume your concerns would be resolved with the use of waypoints. 

@rautamiekka Then we should make it customizable so that a single click on a build icon automatically queries 5 units or something equally out-of-place, because we already dropped consistency out on the window and that's what the RA mod needs.

**Comment by SoScared** [comment]
The entire 4,5 hours is a dedicated competetive 1v1 demonstration on the Playtest.

**Comment by GraionDilach** [comment]
Well, I can easily see that for example the situation around 00:18 is caused by the lack of orange queuing up red's passive buildings via waypoints. If there is a bug with waypoints preventing this to function properly, then that needs to be fixed with priority. That + assault-move should resolve this issue.

**Comment by pchote** [comment]
I think its quite unfair and hurts the discussion to dismiss people's arguments with "competitive discussion is irrelevant", and then respond with a 4.5 hour dedicated competitive stream when asked to provide some solid points backing your own perspective.

**Comment by SoScared** [comment]
No, you don't get to build on your point by saying that players, in this case OpenRA's most quick and adaptive players, play the game wrong thus my point be nullified. This is a classic analytical mistake where you mistake the reality of what is happening to what you want to be happening.

I agree that I can't rest my argument based solely on a link - but I didn't did I? Besides the vod is without a doubt the best possible documentation of the current feature to date so if neither of you have any interest in watching it, now or at any time, or trust that I'm honestly reporting my observations with it then I'm afraid we're in bad shape.

**Comment by pchote** [comment]
I watched the stream live last night, and was surprised at how many veiled positive comments there were on the feature.  Yes, there were complaints, but these were mainly about the frustration at clearing a base - which we now have a plan to address - and at general balance concerns.

Balance is significantly more flexible than core gameplay concepts, and will certainly need to be adjusted to account for this change.  I don't think anybody has claimed otherwise.  I'm not saying that your point is nullified by referring to the competitive players, i'm saying that you are being unfair by dismissing my own points about competitive players and then asserting your own.  It feels like you are constantly shifting the goalposts and changing the argument instead of actually responding to the discussion.

**Comment by GraionDilach** [comment]
I've watched the first half an hour and I believe I've seen enough. I've even came back illustrating my counterpoint with an exact timestamp. (I never got any response on the state of waypoints though, think I even mentioned those in the topic as well.)

I always stated I don't care about actual balance in the mods and that still stands. You're deliberately mixing gameplay design and gameplay balance here. I believe you just decided to not listen.

**Comment by SoScared** [comment]
@pchote I was referring to Grayon's comment regarding his point on the gameplay observation. What points did you make regarding competitive players that I ignored? It must have gone past me.

**Comment by SoScared** [comment]
@GraionDilach I'm not deliberately mixing anything to further an agenda and I take great offense to you claiming me being unreasonable just because I don't agree to your points.

**Comment by GraionDilach** [comment]
You still hasn't answered my question nor made a direct counterclaim to any solution I offered. I can't treat your behavior otherwise.

**Comment by SoScared** [comment]
Well there you have goalpost moving live in action.

**Comment by pchote** [comment]
This topic has crossed the line of what I consider to be a reasonable and positive discussion.  It doesn't seem likely that this can be salvaged to a positive outcome, so I am going to stop this here.

@SoScared: I will PM you on IRC my response to your question above, and if you want to respond then the discussion can continue there.
